.github/workflows/release.yml

```diff
@@ -74,7 +74,8 @@ jobs:
         python -m pip install numpy scipy==1.7.3 docutils "pytest<6" sphinx unittest-xml-reporting
         python -m pip install tinycc h5py sphinx pyparsing html5lib reportlab==3.6.6 pybind11 appdirs
         python -m pip install six numba mako ipython qtconsole xhtml2pdf unittest-xml-reporting pylint
-        python -m pip install qt5reactor periodictable uncertainties debugpy
+        python -m pip install qt5reactor periodictable uncertainties debugpy dominate
```

Comment:
At least on windows, this also needs `importlib_resources` and `html2text`

Reply:
OK, yeah, that needs to be done. Will fix when I'm back at work.

Reply:
Fixed in latest push, if you're happy with it, merge it, I think its time.